src/container.ts

```diff
@@ -0,0 +1,77 @@
+import Docker from 'dockerode';
+
+let DockerClient = new Docker();
+
+
+export class Container {
+
+  name: string;
+  image: string;
+  command: any;
+  hostConfig: object;
+  labels: any;
+  entrypoint: string;
+  environment: any;
+
+  constructor (options: object) {
+    let opts:any = options;
```

Comment:
```js
const {name, image, command, hostConfig, labels, entrypoint, environment} = options
```

Reply:
You could define a type as well for `options` to avoid the above. Will help later.

Reply:
Fixed, already used declared types